avatars/processors/inter_record_bounded_range_difference.py

```diff
@@ -322,6 +322,25 @@ def postprocess(self, source: pd.DataFrame, dest: pd.DataFrame) -> pd.DataFrame:
             ].astype(int)
 
         working = working.sort_values(by=["original_order"])
-        working = working[source.columns]
+
+        # remove temp variables
```

Comment:
temp => tmp ?
tmp is one of the only abreviations that everyone can understand, I'd say use this one or none